helpers/format-website.js

```diff
@@ -0,0 +1,5 @@
+function formatWebsite(website) {
+  return website.startsWith('http') ? website : 'http://' + website;
```

Comment:
This ought to append `http://` to the front of a website if it doesn't already start with it (or `https`). I'll take a look and see what's happening.